apps/condo/domains/common/components/Card/CardsContainer.tsx

```diff
@@ -12,9 +13,9 @@ interface CardsContainerProps {
     autosize?: boolean
 }
 
-const getColSpan = (breakpoints: Partial<Record<Breakpoint, boolean>>) => {
-    if (breakpoints.xxl) return 8
-    if (breakpoints.xl) return 12
+const getColSpan = (breakpoints: Partial<ScreenMap>) => {
```

Comment:
ScreenMap is `Partial` already
Does it really need again?

`export type ScreenMap = Partial<Record<Breakpoint, boolean>>`